node_groups.tf

```diff
@@ -114,6 +114,14 @@ resource "aws_eks_node_group" "node_group" {
     max_unavailable_percentage = lookup(each.value, "update_unavailable_percent", 50)
   }
 
```

Comment:
For adoption upstream, this is probably something we should parameterize, and default to the existing behavior.